modules/custom/d_commerce_products_list/src/Plugin/Block/CommerceMobileFilters.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace Drupal\d_commerce_products_list\Plugin\Block;
+
+use Drupal\Core\Block\BlockBase;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Provides a 'CommerceMobileFilters' Block.
+ *
+ * @Block(
+ *   id = "commerce_mobile_filters",
+ *   admin_label = @Translation("Commerce Mobile filters"),
+ *   category = @Translation("Facets"),
+ * )
+ */
+class CommerceMobileFilters extends BlockBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function blockForm($form, FormStateInterface $form_state) {
+    $form['button_text'] = [
+      '#type' => 'textfield',
+      '#title' => t('Button text'),
+      '#default_value' => $this->configuration['button_text'] ?? 'Filters',
```

Comment:
Jeśli w obrębie jakiejś klasy dostępny jest getter, uzywałbym gettera `$this->getConfiguration()`
To samo w sumie poniżej.